README.md

```diff
@@ -14,3 +14,21 @@ When you open the project for the first time a configuration window pops-up.
 Then click in **Configure Project** and after that **Build** the project.
 
 
+About the BOOST UNITS library:
```

Comment:
Not a big deal, but you can use the markdown syntax to make things better (it's always good to know for further projects).
https://guides.github.com/features/mastering-markdown/